blog/macos-rig/index.qmd

```diff
@@ -0,0 +1,310 @@
+
+## TL;DR
+
+In this post, we cover our preferred setup for R on macOS going into 2023. We highlight these awesome tools: [Homebrew](https://brew.sh/), [Git](https://git-scm.com/download/linux), [zsh](https://zsh.sourceforge.io/), [Oh My Zsh](https://ohmyz.sh/#install), [rig](https://github.com/r-lib/rig), the R packages {[usethis](https://usethis.r-lib.org/)} and {[gitcreds](https://gitcreds.r-lib.org/)}, and [Quarto](https://quarto.org).
+
+If you prefer to look at a short list than read through a long-form blog post, much of the information below is summarized in this [dotfiles](https://github.com/gvelasq/dotfiles/tree/main/macos) repository. Keeping your own [dotfiles](https://dotfiles.github.io) repository, or bookmarking this post or [others](https://www.garrickadenbuie.com/blog/setting-up-a-new-macbook-pro), will help you stay sane next time you are setting yourself up for data science in R on a brand new macOS installation.
+
+- [The Problem](#the-problem)
+- [Installation](#installation)
+- [Final Thoughts](#final-thoughts)
+- [Related Work](#related-work)
+
+## The Problem
+
+Like a new year 🗓️, a new computer 🖥️ (or a fresh reinstall) can bring so many opportunities. A clear desktop! So much memory! And then you remember that you have to install and configure all of your programs again, and that initial excitement gives way to despair and hopelessness. 😱
+
+Okay, that’s a bit of an exaggeration, 🤭 but it can be tedious to reconfigure your setup the way that you like it. However, by using 'dotfiles,' or hidden configuration files (whose names start with a dot/period, which on macOS hides them from view in the Finder), you can store your user preferences on a hosted version control service like GitHub. By dropping dotfiles in the correct directory, you can access and reuse your exact configuration and settings across machines, saving time and hassle.
+
+In this post, we’ll share some helpful dotfiles for all you RStudio users who are rocking a new macOS 🍎installation in 2023. This post is geared towards you if:
+
+* You’ve been installing R and RStudio on macOS and setting them up in your own way already, but either forget the many steps (Git, etc.) or are always having to relearn them;
+* You do a lot of point-and-click downloading of files for installation, and have been considering jumping into command-line adventures 👩🏻‍💻 with Homebrew;
+* You’ve never heard of rig;
+* You want a one-stop-shop for setting up your rig next time;
+* You have a well-oiled, well-defined workflow but are interested in seeing how others set up their machines, to perhaps learn something new!
+
+Since it is the year of [ChatGPT](https://openai.com/blog/chatgpt/), [we did ask its suggested setup](#chat-gpt). We thought we could do better (at least in 2022, maybe not in 2024)! 🤗
+
+Let’s fall into a pit of success <sup>TM</sup> (when configuring macOS)!
```

Comment:
For 'pit of success', add hyperlink to: https://blog.codinghorror.com/falling-into-the-pit-of-success